gridscale/resource_gridscale_k8s.go

```diff
@@ -241,67 +241,57 @@ func resourceGridscaleK8s() *schema.Resource {
 				Optional:    true,
 				Elem:        &schema.Schema{Type: schema.TypeString},
 			},
-			"oidc": {
-				Type:        schema.TypeList,
+			"oidc_enabled": {
```

Comment:
Is there a specific reason why you switched away from OIDClist block? When I implemented I also went without list block at first, but I recognized that for the `node_pool` property you also went with list block instead having properties prefixed with `node_pool_*`. This is why I decided to go with list block also for all properties that belong to OIDC.